Core/src/Detector/VolumeStructureBuilder.cpp

```diff
@@ -175,8 +175,9 @@ Acts::Experimental::VolumeStructureBuilder::construct(
     default:
       break;
   }
+
   // Return the transform, the volume bounds, and some default portal
   // generators
   return {Transform3(m_cfg.transform * eTransform), std::move(volumeBounds),
-          defaultPortalGenerator()};
+          makePortalGenerator<const DefaultPortalGenerator>()};
```

Comment:
Just force `const`?

Reply:
what do you mean by force? I wondered if I should add it to the helper directly. but at the same time I wondered why we require const because the method being called is const anyways :thinking: